basket/news/tasks.py

```diff
@@ -951,11 +966,21 @@ def process_donation(data):
             )
         ):
             sfdc.update(user_data, contact_data)
+            ctms_data = contact_data.copy()
+            del ctms_data["_set_subscriber"]
+            ctms.update(user_data, ctms_data)
     else:
         contact_data["token"] = generate_token()
         contact_data["email"] = data["email"]
         contact_data["record_type"] = settings.DONATE_CONTACT_RECORD_TYPE
 
+        ctms_data = contact_data.copy()
+        del ctms_data["_set_subscriber"]
+        del ctms_data["record_type"]
```

Comment:
CTMS doesn't have these fields. We do have the ``mofo`` group data:

```json
  "mofo": {
    "mofo_email_id": "some-sf-id",
    "mofo_contact_id": "some-other-sf-id",
    "mofo_relevant": false
  }
```

It is possible that we should be setting ``mofo_relevant`` for records that come in this way.

Reply:
I would say "yes", but this decision should likely come from a CTMS decision maker. `record_type` is a SFDC-specific thing that tells it what kind of object to create.